libraries/Microsoft.Bot.Schema.Teams/Generated/MessagingExtensionResponse.cs

```diff
@@ -1,4 +1,4 @@
-// <auto-generated>
+﻿// <auto-generated>
```

Comment:
Presumably you didn't intend to touch this generated code? (remove a blank line and add a namespace)

Reply:
removed